docs/compilation.rst

```diff
@@ -185,46 +187,100 @@ Prerequisites
     ./configure && make
 
 
-Get the DALI source
-+++++++++++++++++++
+Build DALI
+^^^^^^^^^^
+
+# Get DALI source code:
 
 .. code-block:: bash
 
-  git clone --recursive https://github.com/NVIDIA/dali
-  cd dali
+  git clone --recursive https://github.com/NVIDIA/DALI
+  cd DALI
 
-Make the build directory
-++++++++++++++++++++++++
+# Create a directory for CMake-generated Makefiles. This will be the directory, that DALI's built in.
 
 .. code-block:: bash
 
   mkdir build
   cd build
 
+# Run CMake. For additional options you can pass to CMake, refer to OptionalCmakeParamsAnchor_.
+
+.. code-block:: bash
 
-Compile DALI
-^^^^^^^^^^^^
+  cmake -D CMAKE_BUILD_TYPE=Release ..
 
-Building DALI without LMDB support:
-+++++++++++++++++++++++++++++++++++
+# Build. You can pass `-j` option to execute it in several threads
 
 .. code-block:: bash
 
-  cmake ..
   make -j"$(nproc)"
 
+.. _BuildWheelAnchor:
 
-Building DALI with LMDB support:
-++++++++++++++++++++++++++++++++
+Build DALI wheel
+++++++++++++++++
+
+In order to run DALI using Python API, you need to put together and install a wheel based on the binaries you just built.
 
 .. code-block:: bash
 
-  cmake -DBUILD_LMDB=ON ..
-  make -j"$(nproc)"
+  cd build
+  pip wheel dali/python
+  pip install --force-reinstall nvidia_dali-$(cat ../VERSION)-py3-none-any.whl
+
+.. note::
+
+  Since such whl doesn't contain all the dependencies, it will most probably work only on the system where you built DALI bare-metal. To build a wheel that contains the dependencies and might be therefore used on other systems, follow DockerBuilderAnchor_.
```

Comment:
It is a lie. The wheel which is built it not appropriate (self-contained) and doesn't advise to do so. We should stick to only "Install Python bindings" because this is what it is.
I would even stress out that issuing `pip wheel dali/python` is not recommended and won't work as it should.

Reply:
Fixed